app/assets/stylesheets/components/text-quiet.scss

```diff
@@ -0,0 +1,3 @@
+.text__quiet {
+  color: govuk-colour("dark-grey");
```

Comment:
Can we check with @titlescreen to see if there isn existing bit of CSS in the design system we can use, rather than introducing our own custom one?

Reply:
I did, and (unless he misunderstood my question) there isn't. I've put a shout in X-Gov Slack as well.